chpl/chpl-service/src/test/java/gov/healthit/chpl/validation/listing/reviewer/edition2015/RequiredAndRelatedCriteriaReviewerTest.java

```diff
@@ -267,13 +267,13 @@ public void review_a6RemovedCriteriaAttestedWithoutDependencies_asAdmin_hasWarni
                         .build())
                 .build();
         reviewer.review(listing);
-        assertEquals(6, listing.getWarningMessages().size());
-        assertTrue(listing.getWarningMessages().contains(String.format(CRITERIA_COMPLEMENT_NOT_FOUND, "170.315 (a)(*)", Util.formatCriteriaNumber(d1))));
-        assertTrue(listing.getWarningMessages().contains(String.format(CRITERIA_COMPLEMENT_NOT_FOUND, "170.315 (a)(*)", Util.formatCriteriaNumber(d2))));
-        assertTrue(listing.getWarningMessages().contains(String.format(CRITERIA_COMPLEMENT_NOT_FOUND, "170.315 (a)(*)", Util.formatCriteriaNumber(d3))));
-        assertTrue(listing.getWarningMessages().contains(String.format(CRITERIA_COMPLEMENT_NOT_FOUND, "170.315 (a)(*)", Util.formatCriteriaNumber(d5))));
-        assertTrue(listing.getWarningMessages().contains(String.format(CRITERIA_COMPLEMENT_NOT_FOUND, "170.315 (a)(*)", Util.formatCriteriaNumber(d6))));
-        assertTrue(listing.getWarningMessages().contains(String.format(CRITERIA_COMPLEMENT_NOT_FOUND, "170.315 (a)(*)", Util.formatCriteriaNumber(d7))));
+        assertEquals(0, listing.getWarningMessages().size());
```

Comment:
I think the tests need to be renamed as well (just the suffix "_hasWarnings", to maybe "_noWarnings"). Alternatively, do we still need these tests? Maybe they could be removed. But I think the edit is fine.. just one more layer of checking things.